COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.13.0
+3.13.x-40-g93913c7b59-latest
```

Comment:
You shouldn't not add the `-latest` suffix, it will be added by central if slimMode is not used in central